src/components/IssueForm/__tests__/IssueForm.test.tsx

```diff
@@ -0,0 +1,54 @@
+import { cleanup, act } from "@testing-library/react";
```

Comment:
Are you testing the functionality of this page? Like what if everything is ok? You could make a check to see if the submit function is being called with all the required data

Reply:
Yes. It's tested. The form consists of many components and utilities. They are all tested. And for the form itself, I wrote a couple of basic tests